src/index.js

```diff
@@ -16,10 +37,15 @@ export default function (json, colorOptions = {}) {
   json = json.replace(/&/g, '&').replace(/</g, '<').replace(/>/g, '>')
   return json.replace(/("(\\u[a-zA-Z0-9]{4}|\\[^u]|[^\\"])*"(\s*:)?|\b(true|false|null)\b|-?\d+(?:\.\d*)?(?:[eE][+]?\d+)?)/g, (match) => {
     let color = colors.numberColor
+    let style = ''
     if (/^"/.test(match)) {
-      color = /:$/.test(match)
-        ? colors.keyColor
-        : colors.stringColor
+      if (/:$/.test(match)) {
+        color = colors.keyColor
+      } else {
+        color = colors.stringColor;
+        match = '"' + escapeHtml(unescapeJsonString(match.substr(1, match.length - 2))) + '"';
+        style = 'word-wrap:break-word;white-space:pre-wrap;';
```

Comment:
Is it necessary to add the style?

Reply:
Those are for breaking/wrapping long lines inside of <pre></pre>. You can try with a 2000 character long string in a field, check the visual result.

Reply:
Sorry I just thought of this: maybe styles can be put in the pre tag outside, rather than in each of the span tags.  However this needs to be tested.